common/src/main/scala/explore/utils/package.scala

```diff
@@ -66,13 +67,14 @@ package object utils {
     )
   }
 
-  val DefaultPendingRender: Long => VdomNode = _ => Loader(active = true)
+  // TODO All the "potRender" methods should go in lucuma-ui
+  val DefaultPendingRender: VdomNode = Loader(active = true)
 
   val DefaultErrorRender: Throwable => VdomNode = t => Message(error = true)(t.getMessage)
 
   def potRenderWithReuse[A](
     valueRender:   A ==> VdomNode,
-    pendingRender: Long ==> VdomNode = Reuse.always(DefaultPendingRender),
+    pendingRender: => Reuse[VdomNode] = Reuse.always(DefaultPendingRender),
```

Comment:
`Reuse`?

Reply:
I don't think we are using this method anymore. All parameters use `Reuse` here (`A ==> B` is type alias for `Reuse[A => B]`. Since we removed the `Long` for `Pending`, this would be the equivalent.

Anyway, maybe we can remove this function, IDK. It's a utility method that we may want to use some day.